### ISSUE TITLE
Weekly note not detected by calendar

### ISSUE THREAD

**Issue by n-konig** [liamcain/obsidian-periodic-notes#165]
Used the calendar's week number to create a weekly note - as I have previously. The note is created, by does not seem to be detected by the calendar!
Get a question to create the note, and no indicator on the calendar...

Hoping this can be ironed out!

![Cannotfindweeklynote](https://user-images.githubusercontent.com/104557082/210010400-c9007556-ef3e-47aa-8714-89f39f05ae44.jpg)

**Comment by osharp** [comment]
I have the same problem, and it just started today after working fine for months - none of the existing weekly notes are being recognized, and going back to previous months doesn't show the indicator any more next to each week. I'm wondering if it is related to it being the new year and there being some problem with the calendar logic?

**Comment by quanru** [comment]
you need configure in calendar plugin ![image](https://user-images.githubusercontent.com/11739753/211184884-ceb3e374-b19a-4aa9-96ba-ff4eeefabb16.png)

**Comment by n-konig** [comment]
@quanru Thank you, but this isn't the issue. Not sure what versions you're running of the plugin, but these options are not available in mine. Rather, these options are set in the Periodic Notes plugin.
As @osharp seemed to agree, this has not been an issue previously, and as no settings were changed I doubt this is the cause.

**Comment by quanru** [comment]
@n-konig my version is: 
![image](https://user-images.githubusercontent.com/11739753/211231902-6eb16cd0-a079-4588-bb20-fd368e9afe5b.png)

**Comment by n-konig** [comment]
@quanru Right, and as of this version, the plugin author suggests that the weekly note settings are maintained in the Periodic Notes Plugin

> Weekly Notes (deprecated)
> Weekly notes have a new home
> The weekly note functionality has been split out into its [very own plugin](https://github.com/liamcain/obsidian-periodic-notes/). In the future, the functionality will be removed from the Calendar plugin; so if you're currently using weekly notes, I encourage you to make the switch. Don't worry, the behavior is functionally identical and will still integrate with the calendar view!
> This split was inspired by the [One Thing Well](https://en.wikipedia.org/wiki/Unix_philosophy) philosophy. Plugins should be as modular. Some users might want weekly notes and have no use for a calendar view. And vice versa.
> If you are currently using weekly notes within the Calendar plugin, the new Periodic Notes plugin will migrate your settings for you automatically.

🤷

**Comment by dedebenui** [comment]
Same here. It works for my daily notes : [LogBook]/YYYY/MM/[W]WW/YYYY-MM-DD but not for my weekly notes [LogBook]/YYYY/MM/[W]WW/[Review] (it can create it when it doesn't exists, but I get the same error as @devangnegandhi in #166 when trying to open an existing one via the 'open weekly note' command).

I've tried with both 0.0.17 and 1.0.0b3. The only difference with 1.0.0b3 is that I don't get any error when trying to open the existing weekly note : it just doesn't do anything. I've done some tests with other formats and I think the bug happens when the filename is constant (i.e. doesn't have any variable to substitute). For example, when my weekly note format ends in `[Review]` I get the bug, but if I name the file `[W]WW-[Review]` to show the week number in the filename, everything works as expected (in both 0.0.17 and 1.0.0b3).

**Comment by devangnegandhi** [comment]
Ah interesting! I changed my settings for my weekly notes from using `[W]ww` to a static string like `[Weekly Notes]` and I now get the same problem as described here and #166. So @dedebenui you are right - the issue has to do with the constant filename without any variables.

**Comment by n-konig** [comment]
I have the following format for weekly notes: YYYY/gggg-[W]WW
The command "Periodic Notes: Open weekly note" works fine, but the calendar plugin does not recognize last year's notes 😞
